deployment/keystone/changeset/operations/contracts/deploy_contracts_seq.go

```diff
@@ -100,6 +99,16 @@ var DeployKeystoneContractsSequence = operations.NewSequence[DeployKeystoneContr
 			return DeployKeystoneContractsSequenceOutput{}, err
 		}
 
+		// EVM cap OCR3 Contract
+		evmOCR3DeployReport, err := operations.ExecuteOperation(b, DeployOCR3Op, DeployOCR3OpDeps(deps), DeployOCR3OpInput{ChainSelector: input.RegistryChainSelector, Qualifier: "capability_evm"})
```

Comment:
maybe we should pass to `DeployKeystoneContractsSequenceInput` information whether these 2 extra OCR3 contracts (evm and vault) should be deployed? So that we deploy them only when needed?